File: src java/AB.java
public class AB { // А*В без умножения
    public static void main(String[] args) {
        int a = 5; // что умножаем
        int b = 5; // на что умножаем
        int c = a;
        int cycle = 1;
        while (cycle < b) {
            a = a + c;
            System.out.println("Цикл: " + a);
            cycle++;
        }
        System.out.println("-----------");
        System.out.println("Итог: " + a);

    }
}

File: src/Ab.kt
// А*В без умножения
class Ab {
    fun abKotlin() {
        var a = 5
        var b = 5
        var c = a
        var cycle = 1
        while (cycle < b) {
            a += c
            println("Цикл: $a")
            cycle++
        }
        println("----------")
        println("Итог: $a")
    }
}